### ISSUE TITLE
We appear to call `Recording.getRecordingDescription` too early

### ISSUE THREAD

**Issue by loganfsmyth** [RecordReplay/devtools#2820]
If I'm missing something then let me know, but looking over https://github.com/RecordReplay/devtools/blob/cc9ccc4894d07880bd3c0f3984de842a63925323/src/ui/actions/timeline.ts#L54, we call that before the session is created, and thus also before we've guaranteed that the recording itself has finished uploading.

Can we move that call to after session creation, or at least after run `Internal.setAccessToken`?

**Comment by loganfsmyth** [comment]
Ah I missed https://github.com/RecordReplay/devtools/blob/cc9ccc4894d07880bd3c0f3984de842a63925323/src/protocol/thread/thread.ts#L1002

This logic is all pretty weird. If waiting until after session creation is something we're explicitly trying to avoid, we should just make `Recording.getDescription` aware of in-progress recordings. We'd still want to run it `Internal.setAccessToken` though.

**Comment by jasonLaster** [comment]
@loganfsmyth is this still relevant?

**Comment by loganfsmyth** [comment]
Definitely. We should change the frontend to only send `Recording.getDescription` after `Recording.createSession` ideally.
